app/helpers/search_helper.rb

```diff
@@ -183,12 +183,12 @@ def value_for_sign_attribute(number, attribute, main)
   end
 
   def classes_for_sign_attribute(attribute, main)
-    classes = 'image rounded'
-    if main
-      classes += 'main_image'
-    elsif attribute == :handshape
-      classes += 'transition'
-    end
+    # a space is required after the base class names below to ensure that they are
+    # properly separated from other classes assigned in the conditionals.
+    classes = 'image rounded '
+    classes += 'main_image' if main
+    classes += 'transition' if attribute == :handshape
+
```

Comment:
is `main_imagetransition` a class? You'd get that if main && attribute== :handshape

How about assembling an array, and joining it with spaces?
```
classes = ['image', 'rounded']
classes << 'main_image' if main
classes << 'transition' if attribute == :handshape
classes.join(' ')
```

Reply:
oh yeah I didn't consider both of them being added. Thats a nice way of doing it! I'll update the branch now